Mathlib/Geometry/Manifold/DerivationBundle.lean

```diff
@@ -15,8 +15,8 @@ Moreover, we define the differential of a function in terms of derivations.
 
 The content of this file is not meant to be regarded as an alternative definition to the current
 tangent bundle but rather as a purely algebraic theory that provides a purely algebraic definition
-of the Lie algebra for a Lie group.
-
+of the Lie algebra for a Lie group. This theory only coincides with the usual tangent bundle in the
+case of `C^∞` real manifolds.
```

Comment:
Similarly here, "finite-dimensional".

Reply:
Ah, and the use of "only" may imply `C^∞` real as a necessary condition when it is only sufficient.